src/components/NavBar/SuggestionRow.tsx

```diff
@@ -191,18 +186,20 @@ export const TokenRow = ({ token, isHovered, setHoveredIndex, toggleOpen, index,
       </Row>
 
       <Column className={styles.suggestionSecondaryContainer}>
-        {token.market?.price?.value && (
-          <Row gap="4">
-            <Box className={styles.primaryText}>{formatUSDPrice(token.market.price.value)}</Box>
-          </Row>
-        )}
-        {token.market?.pricePercentChange?.value && (
-          <PriceChangeContainer>
-            <ArrowCell>{arrow}</ArrowCell>
-            <PriceChangeText isNegative={token.market.pricePercentChange.value < 0}>
-              {Math.abs(token.market.pricePercentChange.value).toFixed(2)}%
-            </PriceChangeText>
-          </PriceChangeContainer>
+        {!!token.market?.price?.value && (
```

Comment:
`!!` seems unclear to me. are we doing it because the value can be `0` which is falsy? is there a clearer way to express that, or should I just get with the program?

Reply:
Yeah, had to add it for the 0 issue. Boolean(token.market?.price?.value) doesn't appease the typechecker 😢 and creating a sep var for this seems like too much